### PR TITLE
Setup client application

### PR DESCRIPTION
**Pull request by Noggling**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes: [AB#32685](https://statoil-proview.visualstudio.com/9949ad5e-7d15-4531-901e-296574079ee1/_workitems/edit/32685)

## Type of change

Remove options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code.
- [x] I have linked my DevOps task using the AB# tag.
- [x] My code is easy to read.
- [x] Documentation and comments is added where needed.
- [ ] My code is covered by tests.
- [ ] UX has reviewed relevant UI contributions.

### CODE REVIEW

File: client/packages/portal-ui/src/lib/appLoader/appLoader.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
+import { useEffect, useRef } from 'react';
+import { AppLoaderService } from './appLoaderModule';
+
+const AppLoader = new AppLoaderService();
```
Comment (Gustav-Eikaas) [inline client/packages/portal-ui/src/lib/appLoader/appLoader.tsx:4]: appLoader

File: client/packages/portal-ui/src/lib/router/router.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+            <Link to="/page-2">Page 2</Link>
+          </li>
+        </ul>
+      </div>
+      <Routes>
+        <Route
+          path="/"
+          element={
+            <div>
+              This is the generated root route.{' '}
```
Comment (Gustav-Eikaas) [inline client/packages/portal-ui/src/lib/router/router.tsx:22]: {' '}

File: client/packages/portal-ui/src/lib/styleProvider/gobalStyles.ts [hunk 3]
```diff
@@ -0,0 +1,45 @@
+import { tokens } from '@equinor/eds-tokens';
```
Comment (Gustav-Eikaas) [inline client/packages/portal-ui/src/lib/styleProvider/gobalStyles.ts:1]: typo in filename

File: client/packages/portal-ui/src/lib/styleProvider/styleProvider.tsx [hunk 4]
```diff
@@ -0,0 +1,19 @@
+import { EdsProvider, Icon } from '@equinor/eds-core-react';
+import * as icons from '@equinor/eds-icons';
+import { GlobalColorStyle } from './globalColors';
+import { GlobalStyle } from './gobalStyles';
+
+Icon.add(icons);
```
Comment (Gustav-Eikaas) [inline client/packages/portal-ui/src/lib/styleProvider/styleProvider.tsx:6]: consider adding the icons you use

**Review by Gustav-Eikaas — APPROVED** [review]

File: client/packages/portal-client/src/assets/createPortalFramework/createPortalFramework.tsx [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+
+    config.configureServiceDiscovery(portalConfig.serviceDiscovery);
+
+    config.configureMsal(portalConfig.masal.client, portalConfig.masal.options);
+
+    portalConfig.agGrid &&
+      config.addConfig(configureAgGrid(portalConfig.agGrid));
+
+    config.onConfigured(() => {
+      console.log('framework config done');
```
[the remark this answers is not in the document]
Comment (Noggling) [reply client/packages/portal-client/src/assets/createPortalFramework/createPortalFramework.tsx:26]: Will be done when configuring for new portal

File: client/packages/portal-ui/src/lib/styleProvider/styleProvider.tsx (hunk 4 above)
Comment (Noggling) [reply client/packages/portal-ui/src/lib/styleProvider/styleProvider.tsx:6]: will a add coment to do that and make a list of used ones ;)

**Review by Gustav-Eikaas — APPROVED** [review]

**Pull request merged**